pkgs/development/python-modules/sqlalchemy-utils/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, six
+, sqlalchemy
+, pytest
+, flexmock
+, mock
+, python-dateutil
+, pytz
+}:
+
+buildPythonPackage rec {
+  pname = "SQLAlchemy-Utils";
```

I think we're normalizing pname now so this should be
```suggestion
  pname = "sqlalchemy-utils";
```